ref/general/csec-tls.adoc

```diff
@@ -106,3 +106,15 @@ If a default keystore file was already generated with the password from the `ser
 In this configuration, the server creates the keystore and certificate if it does not exist during SSL initialization.
 The certificate is a self-signed certificate with a validity period of 365 days and a signature algorithm of `SHA256withRSA`.
 The certified name (CN) value of the certificate's `subjectDN` attribute is the host name of the machine where the server is running.
+
+=== Default TLS configuration
+
+The minimal configuration that is needed for TLS is a single keystore element called `defaultKeyStore`.
+When the `defaultKeyStore` element exists in the configuration, Open Liberty builds a TLS configuration that is called `defaultSSLConfig` around it.
```

Comment:
" Open Liberty builds a TLS configuration that is called `defaultSSLConfig` around it." take away "that is" for conciseness.